.github/actions/javascript/bumpVersion/index.js

```diff
@@ -1928,7 +1928,7 @@ class SemVer {
     do {
       const a = this.build[i]
       const b = other.build[i]
-      debug('prerelease compare', i, a, b)
+      debug('build compare', i, a, b)
```

Comment:
@BrtqKr @BartoszGrajdek Why is this change necessary?

Reply:
I think it was a leftover after one of the actions